src/main/java/org/mockbukkit/mockbukkit/MockBukkitExtension.java

```diff
@@ -149,4 +157,33 @@ public void afterAll(ExtensionContext context) throws Exception
 		MockBukkit.unmock();
 	}
 
+	@Override
+	public void handleTestExecutionException(ExtensionContext context, Throwable throwable) throws Throwable
+	{
+		if (throwable instanceof UnimplementedOperationException)
+		{
+			logger.info(HORIZONTAL_DIVIDER);
+			logger.info("\t\t\t\t\uD83D\uDEA7 This feature is not implemented yet. \uD83D\uDEA7");
+			logger.info(HORIZONTAL_DIVIDER);
+			logger.info("");
+			logger.info("To help us out, please open an issue on our GitHub repository with this information at");
+			logger.info("https://github.com/MockBukkit/MockBukkit/issues");
+			logger.info("or consider contributing by submitting a pull request. \uD83D\uDCAA");
+			logger.info("");
+			logger.info("Your support and contributions keep the MockBukkit magic alive.");
+			logger.info("Thank you for your collaboration! \uD83D\uDE4F");
+			logger.info("");
+			logger.info(HORIZONTAL_DIVIDER);
+			logger.info("\t\t\t\t\t\t\uD83D\uDCCB Technical details \uD83D\uDCCB");
+			logger.info(HORIZONTAL_DIVIDER);
+			logger.info(() -> String.format("Paper Version: %s", BuildParameters.PAPER_API_FULL_VERSION));
```

Comment:
How about also adding the Paper commit? Should be easy to obtain with [ServerBuildInfo.gitCommit()](https://jd.papermc.io/paper/1.21.4/io/papermc/paper/ServerBuildInfo.html#gitCommit())

Reply:
At the moment with the ServerBuildInfo we are retrieving the MockBukkit commit, not the paper commit:
https://github.com/MockBukkit/MockBukkit/blob/157715f29ce084e17bfc9dc909b2aab6d879e5e5/src/main/java/org/mockbukkit/mockbukkit/services/ServerBuildInfoMock.java#L69-L73

The MockBukkit commit is already present in line 181:
https://github.com/MockBukkit/MockBukkit/blob/838b39a90205461cd931afc14715631a43c227e5/src/main/java/org/mockbukkit/mockbukkit/MockBukkitExtension.java#L179-L183

Reply:
Yep, now that I think about it, it's probably a better idea to put the paper commit there. I think we need to explicitly define the paper commit in for the paper dependency

Reply:
I'm not sure in how we could do that in a way that this could be automatically updated. Does renovate automatically update it?

Reply:
Me neither, I don't know how relevant it is anyhow